src/renderer/components/modal/confirmation/ConfirmationModal.tsx

```diff
@@ -6,13 +6,20 @@ import { useIntl } from 'react-intl'
 import * as Styled from './ConfirmationModal.styles'
 
 type Props = {
+  visible: boolean
```

Comment:
i believe this might be optional prop here and has a default value as `false` later

Reply:
For a `Modal` component, I think it's important that the caller is required to provide the `visible` prop. If you default it to `false` and the caller doesn't provide it, then the modal will never show up and the caller may be confused. IMO it's a better developer experience for the TS compiler to tell them that they're missing the `visible` prop than them having to realize they forgot to set the prop. What do you think?

Reply:
yep. make sense